pkg/commands/run.go

```diff
@@ -80,6 +80,7 @@ func initFlagSet(fs *pflag.FlagSet, cfg *config.Config, m *lintersdb.Manager, is
 	fs.BoolVar(&oc.PrintIssuedLine, "print-issued-lines", true, wh("Print lines of code with issue"))
 	fs.BoolVar(&oc.PrintLinterName, "print-linter-name", true, wh("Print linter name in issue line"))
 	fs.BoolVar(&oc.UniqByLine, "uniq-by-line", true, wh("Make issues output unique by line"))
+	fs.BoolVar(&oc.SortResults, "sort-results", false, wh("Sort issues by line number"))
```

Comment:
can you add the test for this option? Can expand description a little bit as well.

Reply:
I think, @sayboras means to add the tests in the `test/run_test.go` because of the new config option

Reply:
@SVilgelm @sayboras added, in addition, changed help text.